mitosheet/src/components/Mito.tsx

```diff
@@ -211,7 +212,8 @@ export const Mito = (props: MitoProps): JSX.Element => {
             
             return {
                 ...prevUIState,
-                selectedSheetIndex: newSheetIndex
+                selectedSheetIndex: newSheetIndex,
+                selectedTabType: 'data'
```

Comment:
This is getting triggered when the graph message gets sent and as a result the variable `graphTaskpaneOpen` which is used to set the class on the taskpane is not getting set correctly since it now checks for `selectedTabType`.

Reply:
https://user-images.githubusercontent.com/18709905/158092319-71f11e7a-d412-4e51-99e0-ef36ab36e624.mov